sensitivity_analysis.R

```diff
@@ -21,7 +21,7 @@
 if (!require("pacman")) install.packages("pacman"); library(pacman)
 p_load(SoilR, ggplot2, dplyr, tidyr, soilassessment, deSolve, readr)
 
-
+setwd("~/Stages/ClimateFarmers/soil-modelling/soil-modelling")
```

Comment:
We should find an alternative to this. Generally I work with this in a folder and ensure I am working in the wd, or there is some sort of alternative, but this will fail on other computers!

Reply:
I just deleted this line. Forgot it after my personal test of the script.

Reply:
I don't think you've pushed your removal of this line. You should be able to make another commit and then I can approve :)